repo/currency_definition.go

```diff
@@ -415,3 +419,16 @@ func (c CurrencyDictionary) AsMap() map[string]CurrencyDefinition {
 	}
 	return defCopy
 }
+
+// ConfirmationsPerHour will calculate the no of confirmations in 1 hr
+// this is valid only for a crypto
+func ConfirmationsPerHour(currency string) uint64 {
+	defn, err := AllCurrencies().Lookup(currency)
+	if err != nil || defn.CurrencyType != "crypto" {
+		return 0
```

Comment:
I'm not sure I like `confirmationsForTimeout := contract.VendorListings[0].Metadata.EscrowTimeoutHours * 			uint32(repo.ConfirmationsPerHour(order.Payment.AmountCurrency.Code))` being `0` when the `order.Payment.AmountCurrency.Code` is empty or invalid. This seems like a bug that will hurt us unexpectedly later.

Instead, why don't we do the lookup inside `verifyEscrowFundsAreDisputeable` where we can properly handle the error case of having an invalid currency code. Then, we can make a method that looks like `(CurrencyDefinition) ConfirmationsPerHour() uint64` which will always have a valid response. `NilCurrencyDefinition` will still return `0` but you know the dictionary lookup result will be a valid currency if there's no error.